Makefile

```diff
@@ -8,7 +8,7 @@ IMAGE_BUILD_CMD ?= docker build
 IMAGE_BUILD_EXTRA_OPTS ?=
 IMAGE_PUSH_CMD ?= docker push
 CONTAINER_RUN_CMD ?= docker run
-BUILDER_IMAGE ?= golang:1.23-bookworm
+BUILDER_IMAGE ?= golang:v1.23.4-bookworm
```

Comment:
```suggestion
BUILDER_IMAGE ?= golang:1.23.4-bookworm
```

Reply:
:man_facepalming: thanks for noticing this, fixed